client/src/Components/AntdTable/AntdTableComponent.js

```diff
@@ -134,49 +46,39 @@ class AntdTableComponent extends React.Component {
   };
 
   render() {
-    {
-    }
+    const trips = this.props.tripData;
+    const processedTrips = trips.map(trip => {
+      return {
+        ...trip,
+        startTime: DateTime.fromISO(trip.startTime).toFormat('yyyy LLL dd'),
+        distance: trip.distance?.toFixed(2),
+      };
+    });
+
     let { sortedInfo, filteredInfo } = this.state;
     sortedInfo = sortedInfo || {};
     filteredInfo = filteredInfo || {};
     const columns = [
       {
         title: 'Time',
-        dataIndex: 'time',
-        key: 'time',
-        sorter: (a, b) => (a.date < b.date ? -1 : a.date > b.date ? 1 : 0),
-        sortOrder: sortedInfo.columnKey === 'time' && sortedInfo.order,
+        dataIndex: `startTime`,
+        sorter: (a, b) =>
+          a.startTime < b.startTime ? -1 : a.startTime > b.startTime ? 1 : 0,
+        sortOrder: sortedInfo.columnKey === 'startTime' && sortedInfo.order,
       },
 
       {
-        title: 'Distance',
+        title: 'Distance (KM)',
         dataIndex: 'distance',
-        key: 'distance',
         sorter: (a, b) => a.distance - b.distance,
         sortOrder: sortedInfo.columnKey === 'distance' && sortedInfo.order,
+        filteredValue: filteredInfo.distance || null,
+        onFilter: (value, record) => record.distance.includes(null),
```

Comment:
is this correct? I think you should be using the value

Reply:
This was a mistake i left there